bric.c

```diff
@@ -1375,6 +1388,99 @@ void init_editor(void)
         Editor.screen_rows -= 2; // get room for status bar
 }
 
+void load_config_file(void)
+{
+    struct passwd *user = getpwuid(getuid());
+    char config_file[80];
+    config_file[0] = 0;
+    strcat(config_file, user->pw_dir);
+    strcat(config_file, "/.bricrc");
```

Comment:
It's not possible to access ~/.bricrc using fopen("~/.bricrc", "r") because ~ is a bash feautre which is not avalible in C. 

Config file can be, for example:
set linenumbers true
set indent true
set hl_keyword_type_colour 34